src/scheduler.rs

```diff
@@ -168,6 +168,16 @@ async fn fetch_idle(ctx: &Context, connection: &mut Imap, folder: Config) -> Int
                 return connection.fake_idle(ctx, Some(watch_folder)).await;
             }
 
+            if folder == Config::ConfiguredInboxFolder {
+                if let Err(err) = connection
+                    .store_seen_flags(ctx)
+                    .await
+                    .context("store_seen_flags failed")
+                {
+                    warn!(ctx, "{:#}", err);
+                }
+            }
+
```

Comment:
so store_seen_flags is only done on the configured inbox folder?  In the PR comment you say that it's also on the DeltaChat folder?

Reply:
It's done on the Inbox task, but `store_seen_flags` will select folders as needed. When messages should be marked as Seen on DeltaChat folder, Inbox connection will select DeltaChat folder and mark the message there. Weird, but it's the same as before, all markseen jobs are executed on the Inbox connection, I haven't changed it for now.